assets/scripts/templates/list-view.handlebars

```diff
@@ -0,0 +1,36 @@
+<h2>placeholder for list view</h2>
+<table class="table table-responsive" sytle = "float:right">
+<thead class="thead-default">
+  <tr>
+    <th style= "text-align:left">File Title</th>
+    <th style= "text-align:left">File Link</th>
+    <th style= "text-align:left" colspan="3">Actions</th>
+  </tr>
+  </thead>
+  {{#each uploads as |upload|}}
```

Comment:
Should the `tbody` tag be outside of the `#each`?

I would think the `#each uploads` would only be adding new rows, not an entire table body